packages/web/src/components/modal-radio/ModalRadioItem.tsx

```diff
@@ -68,8 +72,11 @@ export const ModalRadioItem = (props: ModalRadioItemProps) => {
           {icon}
           <span>{title ?? label}</span>
         </Text>
+        {tag ? <Tag className={styles.tag} tag={tag} /> : null}
       </div>
-      {hintText ? <HelpCallout content={hintText} /> : null}
+      {hintContent ? (
+        <HelpCallout icon={hintIcon} content={hintContent} />
```

Comment:
Just confirming we want this to always show (it's outside the collapsible)

Reply:
yes!